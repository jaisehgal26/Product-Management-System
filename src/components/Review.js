import React from "react";
import "../App.css";
import Fade from "react-reveal/Fade";

export default function Review() {
  return (
    <div className="container review_container">
      <Fade bottom>
        <section className="ms-4 my-3">
          <div className="row d-flex justify-content-center">
            <div className="col-md-8 col-xl-6 text-center">
              <h3 className="mb-4">#CLIENT'S REVIEW</h3>
              <h1 className="display-6 mb-4 Playfair">
                More Than 20000+ Customers Trusted Us
              </h1>
            </div>
          </div>

          <div className="row text-center">
            <Fade bottom>
              <div className="col-md-4 mb-5 mb-md-0">
                <div className="d-flex justify-content-center mb-4">
                  <img
                    src="https://mdbcdn.b-cdn.net/img/Photos/Avatars/img%20(1).webp"
                    className="rounded-circle shadow-1-strong"
                    width="150"
                    height="150"
                    alt=""
                  />
                </div>
                <h5 className="mb-3 Playfair">Maria Smantha</h5>
                <h6 className="text-primary mb-3">Profession</h6>
                <hr />
                <p className="px-xl-3">
                  <i className="fas fa-quote-left pe-2"></i>Lots of great
                  pastries. Nice people. Good coffee and a squeaky clean
                  restroom ðŸ™‚ A place with self respect! I like it.
                </p>
                <ul className="list-unstyled d-flex justify-content-center mb-0">
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star-half-alt fa-sm text-warning"></i>
                  </li>
                </ul>
                <hr />
              </div>
            </Fade>
            <Fade bottom>
              <div className="col-md-4 mb-5 mb-md-0">
                <div className="d-flex justify-content-center mb-4">
                  <img
                    src="https://mdbcdn.b-cdn.net/img/Photos/Avatars/img%20(2).webp"
                    className="rounded-circle shadow-1-strong"
                    width="150"
                    height="150"
                    alt=""
                  />
                </div>
                <h5 className="mb-3 Playfair">Lisa Cudrow</h5>
                <h6 className="text-primary mb-3">Profession</h6>
                <hr />
                <p className="px-xl-3">
                  <i className="fas fa-quote-left pe-2"></i>Wow! Super cute
                  place with amazing pastries. The staff was also super nice and
                  welcoming.
                  <br></br>
                  Definitely recommend a trip!
                </p>
                <ul className="list-unstyled d-flex justify-content-center mb-0">
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                </ul>
                <hr />
              </div>
            </Fade>
            <Fade bottom>
              <div className="col-md-4 mb-0">
                <div className="d-flex justify-content-center mb-4">
                  <img
                    src="https://mdbcdn.b-cdn.net/img/Photos/Avatars/img%20(3).webp"
                    className="rounded-circle shadow-1-strong"
                    width="150"
                    height="150"
                    alt=""
                  />
                </div>
                <h5 className="mb-3 Playfair">John Smith</h5>
                <h6 className="text-primary mb-3">Profession</h6>
                <hr />
                <p className="px-xl-3">
                  <i className="fas fa-quote-left pe-2"></i>Such a gem! Not only
                  are the staff and pastries / donuts incredible, you can't beat
                  the prices and supporting local! Get here ASAP!
                </p>
                <ul className="list-unstyled d-flex justify-content-center mb-0">
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="fas fa-star fa-sm text-warning"></i>
                  </li>
                  <li>
                    <i className="far fa-star fa-sm text-warning"></i>
                  </li>
                </ul>
                <hr />
              </div>
            </Fade>
          </div>
        </section>
      </Fade>
    </div>
  );
}
