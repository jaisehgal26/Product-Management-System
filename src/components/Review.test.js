import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Review from "./Review";

test("should render correctly", () => {
  const wrapper = toJson(shallow(<Review />));
  expect(wrapper).toMatchSnapshot();
});
