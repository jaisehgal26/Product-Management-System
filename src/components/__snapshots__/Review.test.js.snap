// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="container review_container"
>
  <Fade
    bottom={true}
  >
    <section
      className="ms-4 my-3"
    >
      <div
        className="row d-flex justify-content-center"
      >
        <div
          className="col-md-8 col-xl-6 text-center"
        >
          <h3
            className="mb-4"
          >
            #CLIENT'S REVIEW
          </h3>
          <h1
            className="display-6 mb-4 Playfair"
          >
            More Than 20000+ Customers Trusted Us
          </h1>
        </div>
      </div>
      <div
        className="row text-center"
      >
        <Fade
          bottom={true}
        >
          <div
            className="col-md-4 mb-5 mb-md-0"
          >
            <div
              className="d-flex justify-content-center mb-4"
            >
              <img
                alt=""
                className="rounded-circle shadow-1-strong"
                height="150"
                src="https://mdbcdn.b-cdn.net/img/Photos/Avatars/img%20(1).webp"
                width="150"
              />
            </div>
            <h5
              className="mb-3 Playfair"
            >
              Maria Smantha
            </h5>
            <h6
              className="text-primary mb-3"
            >
              Profession
            </h6>
            <hr />
            <p
              className="px-xl-3"
            >
              <i
                className="fas fa-quote-left pe-2"
              />
              Lots of great pastries. Nice people. Good coffee and a squeaky clean restroom ðŸ™‚ A place with self respect! I like it.
            </p>
            <ul
              className="list-unstyled d-flex justify-content-center mb-0"
            >
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star-half-alt fa-sm text-warning"
                />
              </li>
            </ul>
            <hr />
          </div>
        </Fade>
        <Fade
          bottom={true}
        >
          <div
            className="col-md-4 mb-5 mb-md-0"
          >
            <div
              className="d-flex justify-content-center mb-4"
            >
              <img
                alt=""
                className="rounded-circle shadow-1-strong"
                height="150"
                src="https://mdbcdn.b-cdn.net/img/Photos/Avatars/img%20(2).webp"
                width="150"
              />
            </div>
            <h5
              className="mb-3 Playfair"
            >
              Lisa Cudrow
            </h5>
            <h6
              className="text-primary mb-3"
            >
              Profession
            </h6>
            <hr />
            <p
              className="px-xl-3"
            >
              <i
                className="fas fa-quote-left pe-2"
              />
              Wow! Super cute place with amazing pastries. The staff was also super nice and welcoming.
              <br />
              Definitely recommend a trip!
            </p>
            <ul
              className="list-unstyled d-flex justify-content-center mb-0"
            >
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
            </ul>
            <hr />
          </div>
        </Fade>
        <Fade
          bottom={true}
        >
          <div
            className="col-md-4 mb-0"
          >
            <div
              className="d-flex justify-content-center mb-4"
            >
              <img
                alt=""
                className="rounded-circle shadow-1-strong"
                height="150"
                src="https://mdbcdn.b-cdn.net/img/Photos/Avatars/img%20(3).webp"
                width="150"
              />
            </div>
            <h5
              className="mb-3 Playfair"
            >
              John Smith
            </h5>
            <h6
              className="text-primary mb-3"
            >
              Profession
            </h6>
            <hr />
            <p
              className="px-xl-3"
            >
              <i
                className="fas fa-quote-left pe-2"
              />
              Such a gem! Not only are the staff and pastries / donuts incredible, you can't beat the prices and supporting local! Get here ASAP!
            </p>
            <ul
              className="list-unstyled d-flex justify-content-center mb-0"
            >
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="fas fa-star fa-sm text-warning"
                />
              </li>
              <li>
                <i
                  className="far fa-star fa-sm text-warning"
                />
              </li>
            </ul>
            <hr />
          </div>
        </Fade>
      </div>
    </section>
  </Fade>
</div>
`;
